db/schema.rb

```diff
@@ -83,11 +83,14 @@
     t.integer "role", default: 0
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.bigint "merchant_id"
```

Comment:
Question (I haven't found a clear answer): is it necessary to set the `merchant_id` reference column with a `null: true` value in addition to the `optional: true` set in the model? Or does that automatically ensure that null values are okay if a `user` doesn't `belongs_to` `merchant`?

Reply:
This is something I wanted to get clarification on tomorrow from Mike/Meg.  The users can be created without a merchant I think because of how the model is set up with merchant as an option field but I'm not sure this is 100% set up correctly or fully tested.  I'm assuming the column can be null by default since it is allowing it to be null in some users